src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1084,6 +1088,61 @@ private BidderResponse applyBidPriceChanges(BidderResponse bidderResponse, BidRe
         return bidderResponse.with(BidderSeatBid.of(updatedBidderBids, seatBid.getHttpCalls(), errors));
     }
 
+    private static AdjustmentsMediaType resolveImpMediaType(String bidImpId, List<Imp> imps, BidType bidType) {
+        switch (bidType) {
+            case banner:
+                return AdjustmentsMediaType.banner;
+            case xNative:
+                return AdjustmentsMediaType.xNative;
+            case audio:
+                return AdjustmentsMediaType.audio;
+            case video:
+                return resolveVideoMediaType(bidImpId, imps);
+            default:
+                throw new IllegalArgumentException("BidType not present for bidderBid");
```

Comment:
Actually it's just for check style(switch default), because this exception is not possible at any case.
But replaced  IllegalArgumentException with PreBidException